examples/private_endpoint/variables.tf

```diff
@@ -0,0 +1,7 @@
+variable "prefix" {
+  description = "The Prefix used for all resources in this example"
+}
+
+variable "location" {
+  description = "The Azure Region in which all resources in this example should be created."
+}
```

Comment:
Could we add a sensible default location here?

Reply:
Sensible to me would be the location of the resource group.
But, it doesn't look to me as if that's SOP when I compare with the documentation of other resources like azurerm_virtual_machine.
So, I'm used to having to explicitly define what should have been the default like this:
```
resource "azurerm_virtual_network" "vnt" {
  resource_group_name = azurerm_resource_group.nwk.name
  location            = azurerm_resource_group.nwk.location
....
}
```